### PR TITLE
Remove broken 'user' scope from IrfanSkiljan.IrfanView

### PR DESCRIPTION
**Pull request by treysis**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/77562)

### CODE REVIEW

File: manifests/i/IrfanSkiljan/IrfanView/4.60/IrfanSkiljan.IrfanView.installer.yaml [hunk 1]
```diff
@@ -112,28 +112,14 @@ FileExtensions:
 ReleaseDate: 2022-03-18
 Installers:
 - Architecture: x64
-  Scope: machine
```
Comment (OfficialEsco) [inline manifests/i/IrfanSkiljan/IrfanView/4.60/IrfanSkiljan.IrfanView.installer.yaml:115]: the Machine scopes should stay tho as it always installs for all users

**Review by Trenly — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/77562#discussion_r961853432

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/77562#pullrequestreview-1095235553

**Comment by ItzLevvie** [comment]
Close with reason: Will be fixed in a newer PR;

**Pull request closed without merging**